### ISSUE TITLE
Mip-Levels issue in vk_layer_validation_tests: FramebufferCreateErrors

### ISSUE THREAD

**Issue by CaffeinePwrdAl** [KhronosGroup/Vulkan-LoaderAndValidationLayers#1639]
I've noticed running this test that there is an error in the 'imageView with 2 mip levels' case.

The image you are creating only has one mip level, you then try to create an imageView with 2 levels - this hits an invalid behaviour assert in our Vulkan driver and terminates execution of the tests.

See VkImageObj::init_no_layout in vkrenderframework.cpp:739:

```
VkImageCreateInfo imageCreateInfo = vk_testing::Image::create_info();
imageCreateInfo.imageType = VK_IMAGE_TYPE_2D;
imageCreateInfo.format = fmt;
imageCreateInfo.extent.width = w;
imageCreateInfo.extent.height = h;
imageCreateInfo.mipLevels = 1;     <<<<<<<<
imageCreateInfo.tiling = tiling;
imageCreateInfo.initialLayout = VK_IMAGE_LAYOUT_UNDEFINED;
```

Regards,

Alex

**Comment by jeremy-lunarg** [comment]
It looks like the valid usage is missing for levelCount, which means we send it on down to the driver. I'll add it. Thanks!

**Comment by jeremy-lunarg** [comment]
This should be fixed with #1651. Thanks!
